packages/common/src/streams.ts

```diff
@@ -38,6 +38,24 @@ export const streamToBytes = async (stream: Readable): Promise<Uint8Array> => {
   return new Uint8Array(Buffer.concat(bufs))
 }
 
+const pushIterableToStream = async (
+  iter: AsyncIterable<Uint8Array>,
+  stream: Readable,
+) => {
+  for await (const bytes of iter) {
+    stream.push(bytes)
+  }
```

Comment:
If this throws, I think we should ensure `stream` gets destroyed here.

Reply:
Actually, `Readable.from()` may even do all this for us.

Reply:
wow nice i searched around for that exact fn & didn't find it 🙃